Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG VERSION=unspecified
 
-FROM python:3.12.0-alpine
+# Official Docker images are in the form library/<app> while non-official
+# images are in the form <user>/<app>.
+FROM docker.io/library/python:3.12.0-alpine3.18
```

Comment:
Why not take the opportunity to upgrade to 3.19?  Or are you planning to do that in a later PR?

Reply:
Correct. The `3.12.0-alpine3.18` tag is both equivalent to the previous `3.12.0-alpine` tag and also represents the latest version of Alpine Linux available for a Python 3.12.0 Docker image per [Docker Hub](https://hub.docker.com/_/python/tags?page=1&name=3.12.0-alpine). Since this PR is just about using long names vs. short names I didn't want to bump the version of Python (3.12.1 is the first to be available on Alpine Linux 3.19).